### PR TITLE
UI Login templates and Django routing

### PR DESCRIPTION
**Pull request by cdcabrera**
- UI Django login templates using Patterfly login 
  - Updated ```client/index.html``` to reflect basic views
  - Leaving CDN resource references in place this round
- @quipucords/development separated the routing updates into a second commit, "Django routing updates for login". If needed, can separate these into another PR

updates #434

### CODE REVIEW

File: quipucords/quipucords/templates/registration/login.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+
+{% block page-title %}
+Red Hat Entitlements Reporting
```
Comment (chambridge) [inline quipucords/quipucords/templates/registration/login.html]: Had my fingers crossed 🤞 **_Scanny McScanny Face_**.

Comment (cdcabrera) [reply quipucords/quipucords/templates/registration/login.html]: lol, localization for the win =)

File: quipucords/client/index.html [hunk 2]
```diff
@@ -1,75 +1,84 @@
...
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a href="/" class="navbar-brand">
+          <img class="navbar-brand-icon" src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/img/logo-alt.svg" alt=""/><img class="navbar-brand-name" src="//cdnjs.cloudflare.com/ajax/libs/patternfly/3.30.1/img/brand-alt.svg" alt="PatternFly Enterprise Application" />
+        </a>
+      </div>
+      <nav class="collapse navbar-collapse">
+        <ul class="nav navbar-nav navbar-right navbar-iconic navbar-utility">
```
Comment (jeff-phillips-18) [inline quipucords/client/index.html]: What is all the nav stuff here?

File: quipucords/client/index.html [hunk 3]
```diff
@@ -1,75 +1,84 @@
...
-        </div>
+    <div class="container-fluid container-cards-pf container-pf-nav-pf-vertical nav-pf-persistent-secondary">
+      <div class="row">
+        Lorem ipsum content...
       </div>
     </div>
 
+    <script>
+      $(document).ready(function() {
+        $().setupVerticalNavigation(true);
```
Comment (jeff-phillips-18) [inline quipucords/client/index.html]: Why are we using the jquery vertical navigation?

File: quipucords/client/index.html (hunk 2 above)
Comment (cdcabrera) [reply quipucords/client/index.html]: Most of this stuff is just placeholder. Copied it from  Patternfly site.

File: quipucords/client/index.html (hunk 3 above)
Comment (cdcabrera) [reply quipucords/client/index.html]: Same, placeholder bits. Copied from the Patternfly site.

**Pull request merged**